### PR TITLE
Adding sky color control to semantic segmentation

### PR DESCRIPTION
**Pull request by JonathanHUnity**
# Peer Review Information:
This change adds a sky color to the semantic segmentation labeler.
Note that when this color is changed from the default black, it is no longer considered "background" by the visualizer. I think this is what users will expect, but open to feedback.

Before:
segmentation_3

After:
segmentation_4

## Editor / Package versioning:
**Editor Version Target**: 2019.4

## Dev Testing:
**Tests Added**: Added tests for sky coloring

**Core Scenario Tested**: Tested on URP and HDRP

**At Risk Areas**: 

## Checklist
- [ ] - Updated docs
- [ ] - Updated changelog
- [ ] - Updated test rail

### CODE REVIEW

File: com.unity.perception/Tests/Runtime/GroundTruthTests/SegmentationGroundTruthTests.cs [hunk 1]
```diff
@@ -63,6 +63,7 @@ public class SegmentationPassTests : GroundTruthTestBase
     {
         static readonly Color32 k_SemanticPixelValue = new Color32(10, 20, 30, Byte.MaxValue);
         private static readonly Color32 k_InstanceSegmentationPixelValue = new Color32(255,0,0, 255);
+        private static readonly Color32 k_SkyValue = new Color32(10, 20, 30, 40);
```
Comment (StevenBorkman) [inline com.unity.perception/Tests/Runtime/GroundTruthTests/SegmentationGroundTruthTests.cs:66]: Does the (10, 20, 30, 40) mean something?

File: TestProjects/PerceptionHDRP/Packages/packages-lock.json [hunk 2]
```diff
@@ -81,7 +81,7 @@
         "com.unity.collections": "0.9.0-preview.6",
         "com.unity.nuget.newtonsoft-json": "1.1.2",
         "com.unity.render-pipelines.core": "7.1.6",
-        "com.unity.simulation.capture": "0.0.10-preview.19",
+        "com.unity.simulation.capture": "0.0.10-preview.20",
```
Comment (StevenBorkman) [inline TestProjects/PerceptionHDRP/Packages/packages-lock.json:84]: I don't have a problem with this, but just wanted to make sure that we wanted to upgrade capture to 20 in this fix, and not just happening by accident.

**Review by StevenBorkman — APPROVED** [review]
Nice job.

File: TestProjects/PerceptionHDRP/Packages/packages-lock.json (hunk 2 above)
Comment (JonathanHUnity) [reply TestProjects/PerceptionHDRP/Packages/packages-lock.json:84]: Yea, good eye. this packages-lock.json is an auto-generated file that should have already been updated. It doesn't actually represent a change in the dependencies.

File: com.unity.perception/Tests/Runtime/GroundTruthTests/SegmentationGroundTruthTests.cs (hunk 1 above)
Comment (JonathanHUnity) [reply com.unity.perception/Tests/Runtime/GroundTruthTests/SegmentationGroundTruthTests.cs:66]: no, just wanted to use unique values for each channel to be super sure that we get the same data in as out.

**Pull request merged**